components/brave_wallet_ui/components/desktop/views/portfolio/portfolio-asset.tsx

```diff
@@ -444,7 +444,7 @@ export const PortfolioAsset = (props: Props) => {
     if (!selectedAsset || !selectedAssetsNetwork) return
 
     const account = accounts
-      .filter((account) => account.coin === selectedAsset.coin)
+      .filter((account) => account.accountId.coin === selectedAsset.coin)
```

Comment:
Just noting here that this will replaced by https://github.com/brave/brave-core/pull/18689.